e3db/client.py

```diff
@@ -889,6 +889,26 @@ def __query(self, query):
         return QueryError("An unexpected response occurred, and no results were returned")
 
     def search(self, query):
+        """
+        Public Method to perform improved search request for E3db records according to the query provided.
+
+        Requires a Search object to perform the Query. Construction and Execution is as follows:
+
+        search_this = Search().Match(records=[some_params,...]).Exclude(writers=[omit_this,...])
+        search_result = client.search(search_this) 
+
+        For more information see the documentation of the Search Object or Tests.
+
+        Parameters
+        ----------
+        query : Search
+            Object that contains the information to search for.
+        
+        Returns
+        -------
+        SearchResult
+            Result of a valid response from the search service.
```

Comment:
Same as other comment, remove Search Service in favor of either "server" or "E3DB"